front/src/applications/operationalStudies/components/MacroEditor/export.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  osrdEditoastApi,
+  type Distribution,
+  type TrainScheduleBase,
+} from 'common/api/osrdEditoastApi';
+import type { AppDispatch } from 'store';
+
+import type { NGEDeleteEvent, NGEEvent, NGEUpdateEvent, TrainrunSectionOperation } from './types';
+
+const defautlPayloads = {
+  constraint_distribution: 'STANDARD' as Distribution,
```

Comment:
Nit: I believe `as Distribution` is unnecessary here.

Reply:
Oh right good point!

Reply:
Done